concepticondata/concepticon.tsv

```diff
@@ -2672,3 +2672,8 @@ ID	GLOSS	SEMANTICFIELD	DEFINITION	ONTOLOGICAL_CATEGORY	REPLACEMENT_ID
 2671	PENIS SHEATH	Clothing and grooming	Sheath to cover a penis	Person/Thing	
 2672	HEADWATERS	The physical world	The source of a river or stream	Person/Thing	
 2673	BIRD OF PARADISE	Animals	A bird of the family Paradisaeidae.	Person/Thing	
+2674	ORAL CAVITY	The body	Cavity that is limited by the lips in front, the cheeks on the side, the soft and hard palate at the top and the floor of the mouth below.	Person/Thing	
+2675	MAN OR MALE	Kinship	Relating to the male sex of humans.	Property	
```

Comment:
So it's as always with semantics: a shaky business, and more some kind of "Fingerspitzengefühl". I never knew what people mean by "man/male", but I guess we have so far assembled quite a few concept lists which have exactly those glosses. So linking them all consistently to the "male person" is at least consequent, and if we understand later what it actually means, we'll be able to change it quickly.